Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0
+FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-1.0
 
 # Mandatory!
 ENV APP probate-orchestrator-service.jar
```

Comment:
this is not needed any more :smile:

Reply:
see https://github.com/hmcts/cmc-claim-store/pull/733

Reply:
Thanks Kevin, made some changes.